conda-recipes/pynion-feedstock/recipe/meta.yaml

```diff
@@ -0,0 +1,33 @@
+{% set version = "0.0.5" %}
+
+package:
+  name: pynion
+  version: {{ version }}
+
+source:
+  url: https://github.com/jaumebonet/pynion/archive/{{ version }}.tar.gz
+  sha256: 6798f9359931fa65490e722e7958ed218c889c26e8f48fe052976391003df37a
+
+requirements:
+  host:
+    - python >=2.7.0,<2.8.0a0
```

Comment:
I would remove the python version requirement and let people use with python 3. Looking at setup.py it should work with python 3.

Reply:
There's some other code in pynion (and StrBioInfo to follow) doesn't actually work in py3, and would need porting effort. I thought about adding auto-generated patches using `2to3`and only applying in the python 3 case, but figure that can be added later if there's any interest